frontend/src/App.tsx

```diff
@@ -245,13 +287,23 @@ function AppContainer() {
 
   localStorage.setItem('sidebar', JSON.stringify({ shrink: isSidebarOpen }));
 
+  React.useEffect(() => {
```

Comment:
This could be encapsulated with the other releases based code?

Reply:
done